content/en/scale-agent/concepts/dynamic-accounts.md

```diff
@@ -0,0 +1,139 @@
+  - `TO_DEACTIVATE`: Indicates there is an instruction to deactivate the account.
+  - `DEACTIVATING`: A request to stop watching the account has been sent to the Scale Agent.
+
+
+### Manual migration flow
+
+Migration of an account is the combination of taking the snapshot from a credential source and then activating the accounts.
+
+```mermaid
+sequenceDiagram
+    actor User
+    participant Plugin
+    participant Service
+
+    User->>Plugin: POST /armory/accounts
+    Plugin->>Plugin: Store in clouddriver.kubesvc_accounts
+    User->>Plugin: PATCH /armory/accounts
+    Plugin->>Service: gRPC AddAccounts
+    Service-->>Plugin: return
+    Plugin->>Plugin: Update status in clouddriver.kubesvc_accounts
+```
+
+| POST | PATCH |
+|:------|:-------|
+| {{< include "scale-agent/curl/post-armory-accounts" >}} | {{< include "scale-agent/curl/patch-armory-accounts" >}}   |
+
+See {{< linkWithTitle "scale-agent/tasks/dynamic-accounts/migrate-accounts.md" >}} for detailed instructions and examples.
+
+**What happens when you initiate a migration request**
+
+* If you **do not** include a `zoneId`, the plugin sends the request to every Clouddriver instance that has a connected Scale Agent service. Each Clouddriver instance subsequently sends the request to all of its connected Scale Agent services in an attempt to find one that can process the request.  The account's `zoneId` is updated to be the one of the Agent service that is able to process the add request for that account.
+* If you **do** provide a `zoneId`, the plugin forwards the request to only those Clouddriver instances with a matching Agent service.
+* The plugin decrypts the `kubeconfig` secret and encodes it in Basee64. Then the plugin adds the content as an attribute of the account.
+* The plugin stores the account data in the `clouddriver.kubesvc_accounts` before sending the accounts to the relevant Scale Agent services for processing.
+
+>To reduce overhead, create the `kubeconfig` with only the minimum requirements.
+
+**What happens when the service receives a request to add accounts**
+
+* After receiving the set of accounts, the Scale Agent service parses each `kubeconfig` and fetches the certificate information from the specified cluster. After the fetch succeeds, the Scale Agent service initiates a process that discovers every Kubernetes kind in the target cluster for initiation of a [Kubernetes watch](https://kubernetes.io/docs/reference/using-api/api-concepts/#efficient-detection-of-changes).
+* Next, the Scale Agent service creates a new gRPC connection as a response to tell the Scale Agent plugin which accounts are now active. The plugin then updates account data in the `clouddriver.kubesvc_accounts` table.
+
+The plugin does not inform you of the operation results. Check for an ACTIVE account state in the `clouddriver.kubesvc_accounts` table by querying the database directly or by calling `/agents/kubernetes/accounts/{accountName}`.
```

Comment:
"The plugin does not inform you of the operation results  because it can be a long operation depending on how many accounts are sent"